static/app/views/performance/traceDetails/content.tsx

```diff
@@ -94,14 +99,65 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceNotFound() {
-    const {meta} = this.props;
+    const {meta, traceEventView, traceSlug, organization, location} = this.props;
 
     const transactions = meta?.transactions ?? 0;
     const errors = meta?.errors ?? 0;
 
     if (transactions === 0 && errors > 0) {
+      const eventView = {...traceEventView};
```

Comment:
You should use the [clone method](https://github.com/getsentry/sentry/blob/master/static/app/utils/discover/eventView.tsx#L719) on the `EventView` for this.

Reply:
Now using the [withColumns()](https://github.com/getsentry/sentry/blob/master/static/app/utils/discover/eventView.tsx#L754) to set up only the required columns, which also makes use of the clone method under the hood